frontend/public/components/chargeback.tsx

```diff
@@ -159,12 +159,22 @@ class ReportsDetails extends React.Component<ReportsDetailsProps> {
   }
 }
 
-const COLS_BLACK_LIST = new Set(['data_start', 'data_end']);
-
-const DataCell = ({name, value}) => {
-  if (_.isFinite(value)) {
+const numericUnits = new Set([
+  'bytes',
+  'byte_seconds',
+  'cpu_core_seconds',
+  'cpu_cores',
+  'memory_bytes',
+  'memory_byte_seconds',
+  'seconds']);
+
+const DataCell = ({name, value, unit}) => {
+  if (numericUnits.has(unit)) {
     return <div className="text-right">{_.round(value, 2).toLocaleString()}</div>;
   }
+  if (unit === 'date' || unit === 'time') {
+    return <Timestamp timestamp={value} />;
```

Comment:
Ok, will be handled via: https://bugzilla.redhat.com/show_bug.cgi?id=1739225

Reply:
This needs to be updated to:
```
<Timestamp noRelative={true} timestamp={value} />
```
Once, or after, https://github.com/openshift/console/pull/2341 is merged.

Reply:
This is fixed in https://github.com/openshift/console/pull/2341

Reply:
@chancez, I removed the `noRelative` logic from #2341 because that pr is a fix for our `Timestamp` component, regardless of chargeback report issues.  We need to decide if the fix in #2341 and/or the general `timestamp` displaying of dates is adequate for chargeback report results.

You wrote: "I think It may make sense [to show relative dates] in the details of the report, but i don't think it does in the report results."  

To that end, the fix in #2341 will only show relative dates +/- 10.5 minutes from 'now'.  I guess we need to decide if that is ok for dates in usage report results; particularly `period_start` and `period_end` dates.   If outside the +/- 10.5 minute range, then it will display the dates as "Mmm dd, yyyy t:ms a/p" , if date within current year it will display as "Mmm dd, t:ms a/p" -no year.  Is this ok for dates in report results?   If not, then in a follow up PR we can implement a `fullDate` property for `timestamp` component and update chargeback reports results -thanks

Reply:
I think we want to show the full date for results, always. Otherwise you cannot compare values across rows or cells easily.